### ISSUE TITLE
Clear error messages that should be marked as default

### ISSUE THREAD

**Issue by aanm** [cilium/cilium#13512]
These errors are showed for the first time I enabled `kube-proxy-replacement` with `--device`
```
Oct 12 14:45:38 k8s2 cilium-agent[16875]: level=error msg="Command execution failed" cmd="[tc filter delete dev enp0s8 egress]" error="exit status 2" subsys=datapath-loader
Oct 12 14:45:38 k8s2 cilium-agent[16875]: level=warning msg="Error: Parent Qdisc doesn't exists." subsys=datapath-loader
Oct 12 14:45:38 k8s2 cilium-agent[16875]: level=warning msg="We have an error talking to the kernel, -1" subsys=datapath-loader
```

**Comment by pchaigno** [comment]
I know this error can appear when running Cilium with `--devices` but without `kube-proxy-replacement` or `enable-host-firewall`. #13591 fixes that particular bug.

However, it looks like you're saying you had this error while running **with** `kube-proxy-replacement`?

**Comment by mrostecki** [comment]
I cannot reproduce that with `kube-proxy-replacement` and `device`. I could reproduce that without `kube-proxy-replace` and `device`. So I agree with @pchaigno.

**Comment by pchaigno** [comment]
Yep, we discussed this offline with André and he actually found the issue by running the development VM with devices set. It's the same bug as fixed in #13591.
